purgecss.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "content": ["_site/**/*.html"],
+  "css": ["_site/assets/application*.css"],
+  "whitelist": []
```

Comment:
We ran into some issues with purgecss in the past where we had to chuck a bunch of elements (dropdowns, btns, etc that are created via JS) onto this whitelist. Not the biggest deal but was fairly tedious when Olga would find regressions on legacy pages...

Reply:
I guess tl;dr; do you have any tips for keeping a good whitelist?

Reply:
@lukemoderwell Unfortunately, not a super easy way to do this. It's a tedious and manual process to add whitelisted classes to the config.

The other option you have is to build out a dummy page that has all the markup that isn't getting picked up. If you hide that from the nav but include it in the build, then theoretically those styles should get picked up by purgecss. And if you wanted to take it a step further you could write a hook to delete the file from the build after the build is complete.

That being said, I've found just adding the classes you need to the whitelist is the easiest way to go, as tedious as it may be.